demo/src/screens/componentScreens/SliderScreen.tsx

```diff
@@ -59,41 +59,39 @@ export default class SliderScreen extends Component<SliderScreenProps, SliderScr
               containerStyle={styles.sliderContainer}
               disableRTL
             />
-            <Text bodySmall grey30 style={styles.text}>
+            <Text bodySmall $textNeutral style={styles.text}>
               {sliderValue}%
             </Text>
           </View>
 
-          <Text marginT-30>Negatives</Text>
+          <Text $textDefault marginT-30>
+            Negatives
+          </Text>
           <Slider
             minimumValue={-100}
             maximumValue={100}
             value={-30}
-            minimumTrackTintColor={Colors.red30}
-            thumbTintColor={Colors.red50}
+            minimumTrackTintColor={Colors.$backgroundDangerHeavy}
```

Comment:
These are just colors for the demo, without any contextual meaning. What's the point of migrating them to design tokes? What I mean is that could be any color, red is just one of them...

Reply:
Right, it's not a must.
But since we want the demo screens to look as good as possible in dark mode, we change them to design tokens (where possible), just to make a good contrast with the background.

Reply:
I understand, but I don't see how that makes sense in this context. This is a demo for passing colors to the Slider's track, there is no meaning to the 'danger/warning' tokens here, the red color tints are random. It will be better if you pass 'red60' and 'red80' instead... I don't want us to make this a standard since the screen might look nice but the code doesn't make sense...

Reply:
I see.. 
changed it back to reds